utils/GitUtils.py

```diff
@@ -16,6 +17,16 @@ def get_head_path():
         except (FileNotFoundError, KeyError):
             return None
 
+    @staticmethod
+    def get_current_fork():
+        try:
+            with open(path.join(PathManager.get_git_path(), GitUtils.CONFIG_FILE_NAME), 'r') as git_config_file:
+                git_config_data = str(git_config_file.read())
+                git_config_fork = git_config_data.split('https://github.com/')[1].split('.')[0]
```

Comment:
This can fail because of two reasons (maybe more):
1. There can be multiple remotes.
2. There's no guarantee that the url is going to end with `.git`.

Also, it would be good if you could make it "GitHub independent", since the possibility of it not having a `https://github.com/` url exists (GitLab, self hosted...).

I think you should always grab the `url` of the `origin` remote.

Reply:
Ah yeah definitely, I didn't think of this. Going to make this a draft for now, will rewrite 👍